content/security/rate-limiting.md

```diff
@@ -1,34 +1,79 @@
-### Rate limiting
+### Rate Limiting
 
-A common technique to protect applications from brute-force attacks is **rate-limiting**. Many Express packages exist to provide a rate-limiting feature. A popular one is [express-rate-limit](https://github.com/nfriedly/express-rate-limit).
+In this chapter we cover how to add rate limiting to your application in a more "NestJS Way" than using a package like `express-rate-limit`. To get started, you'll need to install the `@nestjs/throttler` package.
 
-#### Getting started
+```bash
+$ npm i --save @nestjs/throttle
+```
 
-Start by installing the required package:
+Once installation is complete, the `ThrottlerModule` can be configured as any other Nest package with `forRoot` or `forRootAsync`.
 
-```bash
-$ npm i --save express-rate-limit
+```typescript
+@Module({
+  imports: [
+    ThrottlerModule.forRoot({
+      ttl: 60,
+      limit: 10,
+    }),
+  ]
+})
+export class AppModule {}
 ```
 
-Once the installation is complete, apply the rate-limiter as global middleware.
+or
```

Comment:
Following other chapters, can we add a new "Async configuration" section and move this to the end?